### PR TITLE
impl SEND message and a few other minor tweaks

### PR DESCRIPTION
**Pull request by albertocsm**

### CODE REVIEW

**Comment by albertocsm** [comment]
SEND message was impl using Redis itself. A new message is produced to the appropriate topic that will cause all listeners to get it.
Besides making the necessary changes to support the SEND message, also changed the subscription code to try avoiding making a duplicated subscription.

@itzg plz take a look. and let me now what you think.

File: src/main/java/me/itzg/stomprelay/helpers/Listner.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package me.itzg.stomprelay.helpers;
+
+import io.netty.channel.ChannelHandlerContext;
+
+/**
+ * @author @albertocsm
+ */
+public class Listner {
```
Comment (itzg) [inline src/main/java/me/itzg/stomprelay/helpers/Listner.java]: Just a typo, should be "Listener"

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -1,6 +1,11 @@
 spring:
   redis:
-    host: ${redis.host}
+    host: your-hostname
```
Comment (itzg) [inline src/main/resources/application.yml]: A default value of "localhost" would be useful for local development cases. You could also allow for a "short hand" config using `${redis.host:localhost}`

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -1,6 +1,11 @@
...
+    password: "your-password"
```
Comment (itzg) [inline src/main/resources/application.yml]: ...and a default password that's empty would also work well for local development.

File: src/main/resources/application.yml [hunk 4]
```diff
@@ -1,6 +1,11 @@
...
+    port: 0000
```
Comment (itzg) [inline src/main/resources/application.yml]: ...and `${redis.port:6379}`

**Review by itzg — COMMENTED** [review]
This PR is fantastic. As you can tell, I only have been using this as a subscribing-side construct, so the `SEND` capability fills out the true usability of this.

Thanks!

**Comment by albertocsm** [comment]
thks for the input. 

although i don't really see much added value by those "short hand" configs, I'll address all the changes you requested later by the end of the day.

meanwhile, I've pushed a new commit removing some unnecessary code I've added in the previous commits.
prolly its better to just squash this whole thing, in a couple of commits. right?

**Comment by itzg** [comment]
Thanks for the quick updates so far.

Yeah, the shorthand config names and default values are purely selfish, on my part :)

I personally prefer merging the raw commits rather than squashing, so no worries there. I can also do a squash-merge if that works better for you.

**Comment by albertocsm** [comment]
impl the requested changes.
please take a look @itzg

**Review by itzg — APPROVED** [review]
Perfect!

**Pull request merged**